doc/lua_api.txt

```diff
@@ -2590,6 +2591,37 @@ An `InvRef` is a reference to an inventory.
 * `get_location()`: returns a location compatible to `minetest.get_inventory(location)`
     * returns `{type="undefined"}` in case location is not known
 
+### `AreaStore`
+A fast access data structure to store areas, and find areas near a given position or area.
+Every area has a `data` string attribute to store additional information.
```

Comment:
The string doesn't have to be null terminated, you can use raw integers if you want (don't know how to do it with lua). Also, data is optional, can be left out if you want.